src/Ztm.Zcoin.NBitcoin/Exodus/PropertyAmount.cs

```diff
@@ -132,5 +132,15 @@ public static bool operator>(PropertyAmount first, PropertyAmount second)
         {
             return first.value >= second.value;
         }
+
+        public static PropertyAmount Negate(PropertyAmount amount)
+        {
+            if (amount.value == long.MinValue)
+            {
+                throw new OverflowException("The negated value is overflow.");
+            }
+
+            return new PropertyAmount(-amount.Indivisible);
```

Comment:
We should use `value` instead of `Indivisible`.

Reply:
Good idea